lib/src/tar_decoder.dart

```diff
@@ -38,12 +38,10 @@ class TarDecoder {
       // In POSIX formatted tar files, a separate 'PAX' file contains extended
       // metadata for files. These are identified by having a type flag 'X'.
       // TODO: parse these metadata values.
-      if (tf.typeFlag == TarFile.TYPE_G_EX_HEADER ||
-          tf.typeFlag == TarFile.TYPE_G_EX_HEADER2) {
+      if (tf.typeFlag.toLowerCase() == TarFile.TYPE_G_EX_HEADER) {
```

Comment:
Although it's not important because I'm not actually doing anything with PAX data, is there a reason to remove the *_HEADER2 checks? Also, tf.typeFlag.toLowerCase() is being called multiple times, it should be cached in a variable.